Playbooks/Update-CVE-IPs-WatchListwithGreyNoise/azuredeploy.json

```diff
@@ -0,0 +1,1301 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
```

Comment:
metadata requires additional fields. Look at this playbook's metadata section for reference https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/OpenCTI/Playbooks/OpenCTIPlaybooks/OpenCTI-CreateIndicator/azuredeploy.json